catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrProviderTest.java

```diff
@@ -4241,6 +4241,350 @@ public void testSortById() throws Exception {
     }
 
     @Test
+    public void testTextualSort() throws Exception {
+        deleteAllIn(provider);
+
+        List<Metacard> list = new ArrayList<Metacard>();
+
+        DateTime now = new DateTime();
+
+        for (int i = 65; i < 65 + 5; i++) {
+
+            MockMetacard m = new MockMetacard(Library.getFlagstaffRecord());
+
+            m.setEffectiveDate(now.minus(5L * i)
+                    .toDate());
+
+            m.setTitle((char) i + " Record ");
+
+            list.add(m);
+
+        }
+
+        create(list);
+
+        Filter filter = null;
+        QueryImpl query = null;
+        SourceResponse sourceResponse = null;
+
+        // Sort all Textual ASCENDING
+
+        filter = filterBuilder.attribute(Metacard.EFFECTIVE)
+                .before()
+                .date(now.plusMillis(1)
+                        .toDate());
+
+        query = new QueryImpl(filter);
+
+        query.setSortBy(new ddf.catalog.filter.impl.SortByImpl(Metacard.TITLE,
+                SortOrder.ASCENDING.name()));
+
+        sourceResponse = provider.query(new QueryRequestImpl(query));
+
+        assertEquals(list.size(),
```

Comment:
We should be using `assertThat` here (and `hasSize`).  https://codice.atlassian.net/wiki/display/DDF/Unit+Testing